doc/cli.markdown

```diff
@@ -598,7 +598,7 @@ If you want to see config variables, used to configure
 balena features, use the --config option.
 
 At the moment the CLI does not support per-service variables,
-so the following commands will only show service-wide
+so the following commands will only show application-wide
```

Comment:
How about ?
```suggestion
so the following commands will only show application/device-wide
```
```suggestion
so the following commands will only show application-wide or device-wide
```

Reply:
Thanks, I've reworded it to avoid '-wide' altogether. :-)